src/testing/marks.zig

```diff
@@ -85,7 +87,7 @@ pub fn wrap_log(comptime base: anytype) type {
             pub const mark = struct {
                 pub fn err(comptime fmt: []const u8, args: anytype) void {
                     record(fmt);
-                    base.err(fmt, args);
+                    base.warn(fmt, args);
```

Comment:
An alternate fix would be to pass `stdx.log.scoped(...)` in as the base logger to `wrap_log()`.